### DISCUSSION TITLE
How to use toggle_auto_expand_width resize bigger width?

### DISCUSSION THREAD

**Discussion by linrongbin16** [nvim-neo-tree/neo-tree.nvim#716] [Q&A] [answered]
Hi, I'm new to neo-tree.

I want to resize neotree width (by default it's 40), I find there's a command `toggle_auto_expand_width`, but looks like it's resize +10 width when first press `e`, then it's reset to default width when second press `e`.

Is there a way to continue resize bigger/smaller width? For example provide a command like nvim-tree:
```
:NvimTreeResize -10
:NvimTreeResize +10
```

**Comment by cseickel** [comment, marked as answer]
I don't see the point in adding those commands when nvim already has resize commands built-in.

What auto-expand width does is to ensure that the window is always wide enough to show all of the content. It doesn't add 10 unless 10 happens to be the amount of columns needed to show all content.

**Reply by linrongbin16** [reply]
Thanks, I found some tips on resize vim window width here: https://vi.stackexchange.com/a/515/6600
